lib/components/user/monitored-trip/trip-basics-pane.js

```diff
@@ -102,6 +103,7 @@ class TripBasicsPane extends Component {
 
       return (
         <div>
+          <TripStatus monitoredTrip={monitoredTrip} />
```

Comment:
Do we want the trip status to be displayed when the user is saving the itinerary for the first time? (At that point, the trip has not been created in the database.)